module/actor/actor-sheet.js

```diff
@@ -427,11 +436,11 @@ export class VampireActorSheet extends ActorSheet {
     else if (total_crit_success){
       label = label + `<p class="roll-content">${game.i18n.localize('VTM5E.CriticalSuccess')}</p>`;
     }
-    if (hunger_crit_fail){
-      label = label + `<p class="roll-content">${game.i18n.localize('VTM5E.PossibleBestialFailure')}</p>`;
+    if (hunger_crit_fail && !successRoll){
+      label = label + `<p class="roll-content">${game.i18n.localize('VTM5E.BestialFailure')}</p>`;
```

Comment:
if I set the difficulty to 0 (i.e. rolling without knowing what difficulty I want) and roll a possible bestial failure, it'll output only Bestial Failure! which may be confusing to new V5 players